app/templates/race/details.html

```diff
@@ -328,4 +328,26 @@ <h4 class="text-right">
     </pre>
   {%endif%}
 {%endif%}
+<div id="disqus_thread"></div>
+<script>
+
+/**
+*  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+*  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables*/
+/*
+var disqus_config = function () {
```

Comment:
Agreed, I realized I did my testing directly on a staging server and didn't commit my code.